python/cuml/test/test_tsne.py

```diff
@@ -44,7 +44,77 @@ def check_embedding(X, Y, score=0.76):
 
 
 @pytest.mark.parametrize('name', dataset_names)
-def test_tsne(name):
+@pytest.mark.parametrize('type_knn_graph', ['sklearn', 'cuml'])
+@pytest.mark.parametrize('method', ['fft', 'barnes_hut'])
+def test_tsne_knn_graph_used(name, type_knn_graph, method):
+
+    datasets
+    X = eval("datasets.load_{}".format(name))().data
+
+    neigh = skKNN(n_neighbors=90) if type_knn_graph == 'sklearn' \
+        else cuKNN(n_neighbors=90)
+
+    neigh.fit(X)
+    knn_graph = neigh.kneighbors_graph(X, mode="distance").astype('float32')
+    tsne = TSNE(random_state=1,
+                n_neighbors=90,
+                angle=0.8,
+                method=method)
+
+    # Perform tsne with normal knn_graph
+    Y = tsne.fit_transform(X, True, knn_graph)
+    trust_normal = trustworthiness(X, Y)
+    print("Trust = ", trust_normal)
```

Comment:
Please dont print into the test log. These appear in every CI run and make the CI log 40,000+ lines.

Reply:
There are several print statements in this test file that should be removed FYI. I'm just going to have this one comment instead of marking them all.